structure/src/main/scala/com/evolution/controllers/UserController.scala

```diff
@@ -0,0 +1,41 @@
+package com.evolution.controllers
+
+import com.evolution.domain._
```

Comment:
shouldn't we set:
```scala 
      scalacOptions := Seq("-Xsource:3")
```
and amend `.scalafmt.conf` with:
```
runner.dialect = Scala213Source3
```
and here use future-proof `import com.evolution.domain.*`?